ui/src/Components/Store/LoginSice.js

```diff
@@ -0,0 +1,45 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit"
+import axios from "axios"
+let username = '';
+export const logindata  = createAsyncThunk('user/logindata', async (payload)=>{
+    username = payload.username;
+    console.log( "ddd " +payload.username)
+   return axios.post('https://trinitstechnologies.com/demo/api/v1/user/login',payload)
+
+
+
+
+})
+
+const initialState ={
+    username:"",
+    islogdin:false
```

Comment:
Change the variable name to 'isUserLoggedIn'

Reply:
done